### PR TITLE
Add blog post: OTel End User Q&A  Series: OTel with GraphQL

### PR DESCRIPTION
**Pull request by avillela**
Blog post summarizing January 26th OpenTelemetry End User Q&A session.

**Preview**: https://deploy-preview-2300--opentelemetry.netlify.app/blog/2023/otel-end-user-q-and-a-series-otel-and-graphql/

### CODE REVIEW

File: content/en/blog/2023/otel-end-user-q-and-a-series-otel-and-graphql.md [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+title: >
+    End-User Q&A Series: Using OTel with GraphQL
+linkTitle: >
+    End-User Q&A: OTel with GraphQL
+date: 2023-02-07
+author: >-
+  [Adriana Villela](https://github.com/avillela) (Lightstep)
+  <br/> with contributions from: <br/>
+  [Rynn Mancuso](https://github.com/musingvirtual) (Honeycomb) |
+  [Reese Lee](https://github.com/reese-lee) (New Relic)
```
Comment (chalin) [inline content/en/blog/2023/otel-end-user-q-and-a-series-otel-and-graphql.md:11]: Oh, ... could we find another way of acknowledging contributions? Maybe just putting it in the main text either at the end, or at the start (in smaller font and italics)?

**Review by chalin — COMMENTED** [review]
See inline comment.

Also, pls run `npx prettier --ignore-path '' -w <path-to-your-file>` (this will soon be checked automatically).

Thanks!

File: content/en/blog/2023/otel-end-user-q-and-a-series-otel-and-graphql.md (hunk 1 above)
Comment (svrnm) [reply content/en/blog/2023/otel-end-user-q-and-a-series-otel-and-graphql.md:11]: yeah, that looks odd, put it somewhere in an acknowledgment section or just list them as full authors, I did it that way in the past for some of my blog posts (the nginx apache ones)

File: content/en/blog/2023/otel-end-user-q-and-a-series-otel-and-graphql.md [hunk 2]
```diff
@@ -0,0 +1,163 @@
...
+
+Across the organization, different teams have chosen to use different observability platforms to suit their needs, resulting in a mix of both open source and proprietary observability tools.
+
+J’s team had recently migrated from one observability back-end to another. After this migration, they started seeing gaps in trace data, because other teams that they integrated with were still using a different observability back-end. As a result, they no longer had an end-to-end picture of their traces. The solution was to use a standard, vendor-neutral way to emit telemetry: OpenTelemetry.
+
+Another reason that his team took to using OpenTelemetry was [GraphQL](https://graphql.org), which they had been using for four years. [GraphQL](https://en.wikipedia.org/wiki/GraphQL) is an open-source language used to query and manipulate APIs. With GraphQL, everything is held in the body of data: request, response and errors, and as a result everything returns an [HTTP status of 200](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/200), giving the impression that even failures are successful. This meant that J and his team had no visibility into what was going on behind the scenes.
```
Comment (svrnm) [inline content/en/blog/2023/otel-end-user-q-and-a-series-otel-and-graphql.md:42]: I find it odd to link to 2 different places with the same word, I suggest to drop the link to the wikipedia page.

**Review by svrnm — CHANGES_REQUESTED** [review]

File: content/en/blog/2023/otel-end-user-q-and-a-series-otel-and-graphql.md (hunk 2 above)
Comment (chalin) [reply content/en/blog/2023/otel-end-user-q-and-a-series-otel-and-graphql.md:42]: Right. The oddness comes from the fact that the same anchor text is used. I'd vote to keep the link but change the anchor text:

> GraphQL is an [open-source language used to query and manipulate APIs](https://en.wikipedia.org/wiki/GraphQL).

WDYT?

Comment (avillela) [reply content/en/blog/2023/otel-end-user-q-and-a-series-otel-and-graphql.md:42]: I'm good with your suggestion @chalin!

File: content/en/blog/2023/otel-end-user-q-and-a-series-otel-and-graphql.md (hunk 1 above)
Comment (avillela) [reply content/en/blog/2023/otel-end-user-q-and-a-series-otel-and-graphql.md]: Fair point! I've added a line at the beginning of the post like this:

> *with contributions from: [Rynn Mancuso](https://github.com/musingvirtual) (Honeycomb) and [Reese Lee](https://github.com/reese-lee) (New Relic)*

Let me know if that works @svrnm @chalin?

**Comment by chalin** [comment]
This PR needs to be rebased (there's a change in the tooling that could impact this PR), and Prettier run over the post. Do you want to do it @avillela or would you like for me to handle it?

File: content/en/blog/2023/otel-end-user-q-and-a-series-otel-and-graphql.md (hunk 1 above)
Comment (svrnm) [reply content/en/blog/2023/otel-end-user-q-and-a-series-otel-and-graphql.md]: works for me.

File: content/en/blog/2023/otel-end-user-q-and-a-series-otel-and-graphql.md [hunk 3]
```diff
@@ -0,0 +1,162 @@
...
+title: >
+    End-User Q&A Series: Using OTel with GraphQL
+linkTitle: >
+    End-User Q&A: OTel with GraphQL
+date: 2023-02-07
+author: >-
+  [Adriana Villela](https://github.com/avillela) (Lightstep)
+---
+
+*with contributions from: [Rynn Mancuso](https://github.com/musingvirtual) (Honeycomb) and [Reese Lee](https://github.com/reese-lee) (New Relic)*
```
Comment (chalin) [inline content/en/blog/2023/otel-end-user-q-and-a-series-otel-and-graphql.md:11]: Yes, this is better! Here are some tweaks:

```suggestion
*With contributions from [Rynn Mancuso](https://github.com/musingvirtual) (Honeycomb) and [Reese Lee](https://github.com/reese-lee) (New Relic)*.
```

**Comment by avillela** [comment]
> This PR needs to be rebased (there's a change in the tooling that could impact this PR), and Prettier run over the post. Do you want to do it @avillela or would you like for me to handle it?

@chalin I just did a re-base and re-ran prettier. Let me know if there are any issues!

File: content/en/blog/2023/otel-end-user-q-and-a-series-otel-and-graphql.md [hunk 4]
```diff
@@ -0,0 +1,346 @@
...
+some pull requests (PRs) is very slow. One way to mitigate this is to get
+contributors more involved, which could potentially help bring in more
+contributors.
+
+### Documentation
+
+J and his team have also experienced some challenges with documentation, noting
+that there are some gaps in the online docs:
+
+- Under metrics for JavaScript there is no mention of the Observable Guage at
```
Comment (chalin) [inline content/en/blog/2023/otel-end-user-q-and-a-series-otel-and-graphql.md:311]:
```suggestion
- Under metrics for JavaScript, there is no mention of the Observable Guage at
```

File: content/en/blog/2023/otel-end-user-q-and-a-series-otel-and-graphql.md [hunk 5]
```diff
@@ -0,0 +1,346 @@
...
+  all. J had to go into the code to find it.
+- There are some short, very high-level metric API examples. Those examples
+  currently don't show which libraries you need to bring in. It also doesn't
+  talk about how to export items.
+- In .NET it is very hard to keep a trace going in your work due to all the
```
Comment (chalin) [inline content/en/blog/2023/otel-end-user-q-and-a-series-otel-and-graphql.md:316]:
```suggestion
- In .NET, it is very hard to keep a trace going in your work due to all the
```

File: content/en/blog/2023/otel-end-user-q-and-a-series-otel-and-graphql.md [hunk 6]
```diff
@@ -0,0 +1,346 @@
...
+  async/await and it jumping between threads. .NET docs lack some detail around
+  context propagation in this particular scenario.
+
+.NET docs and keeping context. I found a bunch of instrumentation but I noticed
+it has a hard time keeping the context around.
```
Comment (chalin) [inline content/en/blog/2023/otel-end-user-q-and-a-series-otel-and-graphql.md:321]: This paragraph seems odd. The first sentence is incomplete.

**Review by chalin — CHANGES_REQUESTED** [review]
See inline comments regarding the text added via the last commit. Thx

File: content/en/blog/2023/otel-end-user-q-and-a-series-otel-and-graphql.md (hunk 6 above)
Comment (avillela) [reply content/en/blog/2023/otel-end-user-q-and-a-series-otel-and-graphql.md]: > This paragraph seems odd. The first sentence is incomplete.

My bad - that should've been removed. Deleted.

Comment (svrnm) [reply content/en/blog/2023/otel-end-user-q-and-a-series-otel-and-graphql.md]: please "resolve" the conversation

**Review by chalin — APPROVED** [review]
Other than a date change, LGTM. Thx

**Review by svrnm — APPROVED** [review]

**Pull request merged**